src/testUtils/java/uk/gov/hmcts/reform/waworkflowapi/SpringBootFunctionalBaseTest.java

```diff
@@ -2,12 +2,21 @@
 
 import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
 import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
 
 @RunWith(SpringIntegrationSerenityRunner.class)
 @SpringBootTest
-@ActiveProfiles("functional")
+@ActiveProfiles(profiles = {"local","functional"})
```

Comment:
do we need local to run on the pipeline too :) ?

Reply:
removed.